### DISCUSSION TITLE
NotMapped Column disappearing on save?

### DISCUSSION THREAD

**Discussion by rchawla-alphabyte** [serenity-is/Serenity#5961] [Help]
Hi,

I have an embedded grid in my dialog. This grid has an NotMapped column used for a calculation.

The issue is that the column value seems to disappear once the main dialog is saved. Any suggestions on how to keep the NotMapped column value once saved?

Thanks

**Comment by reach2rv** [comment]
Not Mapped fields value are not saved in database so to get this value on retrieve or opening dialog either save those values in database or reassign the values to them in afterload entity function on dialog

**Comment by luongnv6** [comment]
use MinSelectLevel(SelectLevel.List) for NotMapped Field on Row.cs

**Reply by rchawla-alphabyte** [reply]
This does not work for me
